src/index.html

```diff
@@ -4,9 +4,324 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list list">
+          <li class="list__element">
+            <a href="#" class="list__link is-active">Apple</a>
+          </li>
```

Comment:
```suggestion
      <nav class="navigation">
        <ul class="navigation__list">
          <li class="navigation__item">
            <a href="#" class="navigation__link is-active">Apple</a>
          </li>
```

no need extra blocks here, just use nav as a block, and give semantic names for elements:

nav - nav or navigation as u did
ul - unordered list  - nav__list
li - list item - nav__item
a - link - nav__link

just do it simple

Reply:
Done